Backend/Remora.Discord.API/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -1052,7 +1052,9 @@ private static JsonSerializerOptions AddInteractionObjectConverters(this JsonSer
         options.AddDataObjectConverter<IInteractionResponse, InteractionResponse>();
 
         options.AddDataObjectConverter<IApplicationCommand, ApplicationCommand>()
-            .WithPropertyName(d => d.IsNsfw, "nsfw");
+            .WithPropertyName(d => d.IsNsfw, "nsfw")
+            .WithPropertyName(d => d.AllowedContextTypes, "contexts")
```

Comment:
I'm somewhat averse towards renaming these - the only other elements we rename are booleans (to C#-ify them as a question or statement of fact) and fields which don't convert correctly automatically (usually with abbreviations involved).

I'm open to hearing a rationale for this, but it'd have to be pretty convincing. Otherwise, please stick to Discord's naming scheme.

Reply:
At the time of writing this code, it was referred to as "allowed contexts" in the documentation, while the actual field they specified was `contexts`; since this documentation has since changed however, I can change the field as well I suppose